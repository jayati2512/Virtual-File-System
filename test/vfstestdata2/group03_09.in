createvfs vfs1 1000
mountvfs vfs1
makedir / root
makedir /root d1
makedir /root d2
makedir /root/d1 d3
makedir /root/d1 d4
makedir /root/d2 d5
makedir /root/d2 d6
makedir /root/d1/d3 d7
makedir /root/d1/d3 d8
makedir                           
makedir /root/d1 d3              
addfile /root/d1 file1 /usr/include/stab.h
addfile /root/d1 file2 /usr/include/stab.h
addfile /root/d1 file2 /usr/include/stab.h
addfile 
addfile /root/d1/d3 file3 /usr/include/stab.h
addfile /root/d1/d3 file4 /usr/include/stab.h
deletedir /root/d1/d3           
deletedir /root/d2/d5
movedir /root/d2/d6 /root/d1/file1
movedir /root/d2/d6 /root/d1
listfile /root/d1/file1 /tmp/output_listfile.txt
listdir /root/d1 0 /tmp/output_listdir.txt
searchfile file /tmp/output_searchfile.txt
unmountvfs vfs1
