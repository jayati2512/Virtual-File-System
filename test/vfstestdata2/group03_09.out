createvfs_SUCCESS
mountvfs_SUCCESS
makedir_SUCCESS
makedir_SUCCESS
makedir_SUCCESS
makedir_SUCCESS
makedir_SUCCESS
makedir_SUCCESS
makedir_SUCCESS
makedir_SUCCESS
makedir_SUCCESS
makedir_FAILURE VFS_INSUFFICIENT_ARGUMENTS
makedir_FAILURE DIRECTORY_ALREADY_EXISTS
addfile_SUCCESS
addfile_SUCCESS
addfile_FAILURE FILE_ALREADY_EXISTS
addfile_FAILURE VFS_INSUFFICIENT_ARGUMENTS
addfile_SUCCESS
addfile_SUCCESS
deletedir_FAILURE DIRECTORY_IS_NOT_EMPTY
deletedir_SUCCESS
movedir_FAILURE DESTINATION_CANNOT_BE_FILE
movedir_SUCCESS
listfile_SUCCESS
listdir_SUCCESS
searchfile_SUCCESS
unmountvfs_SUCCESS
